docs/ug/cli.md

```diff
@@ -107,6 +107,15 @@ The last modified dates will be in the same timezone specified with the `--timez
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
+### `--shallow-cloning`, `-S`
```

Comment:
This page is arranged alphabetically as of now. So better to push it down accordingly.